src/main/webapp/WEB-INF/tags/instructor/results/responseRow.tag

```diff
@@ -8,87 +8,87 @@
-                        </div>
-                    </c:when>
-                    <c:otherwise>
-                            <div class="align-center" data-link="">
-                                <a class="student-profile-pic-view-link btn-link">
-                                    No Photo
-                                </a>
-                            </div>
-                    </c:otherwise>
-                </c:choose>
-            </td>
-        </c:if>
-        <td class="word-wrap-break middlealign<c:if test="${responseRow.rowGrey}"> color_neutral</c:if>">${fn:escapeXml(responseRow.recipientTeam)}</td>
-        <td class="word-wrap-break middlealign<c:if test="${responseRow.rowGrey}"> color_neutral</c:if>">
+    <td class="word-wrap-break middlealign<c:if test="${responseRow.rowGrey}"> color_neutral</c:if>">${fn:escapeXml(responseRow.giverTeam)}</td>
+    <td class="word-wrap-break middlealign<c:if test="${responseRow.rowGrey}"> color_neutral</c:if>">
+    <c:choose>
+      <c:when test="${not empty responseRow.giverProfilePictureLink && !responseRow.giverProfilePictureAColumn}">
+        <div class="profile-pic-icon-hover" data-link="${responseRow.giverProfilePictureLink}">
+          ${fn:escapeXml(responseRow.giverDisplayableIdentifier)}
+          <img src="" alt="No Image Given" class="hidden profile-pic-icon-hidden">
+        </div>
+      </c:when>
+      <c:otherwise>
+        ${fn:escapeXml(responseRow.giverDisplayableIdentifier)}
+      </c:otherwise>
+    </c:choose>
+    </td>
+  </c:if>
+  <c:if test="${responseRow.recipientDisplayed}">
+    <c:if test="${responseRow.recipientProfilePictureAColumn}">
+      <td class="middlealign<c:if test="${responseRow.rowGrey}"> color_neutral</c:if>">
         <c:choose>
-            <c:when test="${not empty responseRow.recipientProfilePictureLink && !responseRow.recipientProfilePictureAColumn}">
-                <div class="profile-pic-icon-hover" data-link="${responseRow.recipientProfilePictureLink}">
-                    ${fn:escapeXml(responseRow.recipientDisplayableIdentifier)}
-                    <img src="" alt="No Image Given" class="hidden profile-pic-icon-hidden">
-                </div>
-            </c:when>
-            <c:otherwise>
-                ${fn:escapeXml(responseRow.recipientDisplayableIdentifier)}
-            </c:otherwise>
+          <c:when test="${not empty responseRow.recipientProfilePictureLink}">
+            <div class="profile-pic-icon-click align-center" data-link="${responseRow.recipientProfilePictureLink}">
+              <a class="student-profile-pic-view-link btn-link">
+                View Photo
+              </a>
+              <img src="" alt="No Image Given" class="hidden">
+            </div>
+          </c:when>
+          <c:otherwise>
+              <div class="align-center" data-link="">
+                <a class="student-profile-pic-view-link btn-link">
+                  No Photo
+                </a>
+              </div>
```

Comment:
He's talking about the entire `div` section. See lines 24-28 in the latest `responseRow.tag`.